project/accounts/views.py

```diff
@@ -188,6 +188,21 @@ def get(self, request, username=None):
         )
 
 
+class UserFollowing(LoginRequiredMixin, View):
+    """A view that shows list of users following user with given username"""
+
+    def get(self, request, username=None):
+        profile = get_object_or_404(Profile, user__username=username)
+
+        return TemplateResponse(
+            request,
+            "user_civis.html",
```

Comment:
It would be more accurate for this template to be called `profile_followers.html`.